UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ValidateAuthorizationForAPIsWithExchangeSetStandardParameterTests.cs

```diff
@@ -43,12 +44,13 @@ public async Task WhenICallSinceDateTimeApiWithAValidADTokenAnds57AsExchangeSetS
 
         #region ProductIdentifiers Api
         // PBI 140109 : ESS API : Add authorization to allow only UKHO people to create unencrypted ES 
+        //PBI 142740: ESS APIs : UKHO Authorization - Change Unauthorize(401) to Forbidden(403)
         [Test]
         [Category("SmokeTest-AIODisabled")]
         public async Task WhenICallProductIdentifiersApiWithAValidB2cTokenAnds57AsExchangeSetStandardParameter_ThenAnUnauthorizedResponseIsReturned()
         {
             var apiResponse = await ExchangeSetApiClient.GetProductIdentifiersDataAsync(DataHelper.GetProductIdentifierData(), null, accessToken: EssB2CToken, "s57");
-            Assert.AreEqual(401, (int)apiResponse.StatusCode, $"Incorrect status code {apiResponse.StatusCode} is returned, instead of the expected 401.");
+            Assert.AreEqual(403, (int)apiResponse.StatusCode, $"Incorrect status code {apiResponse.StatusCode} is returned, instead of the expected 401.");
```

Comment:
```suggestion
            Assert.AreEqual(403, (int)apiResponse.StatusCode, $"Incorrect status code {apiResponse.StatusCode} is returned, instead of the expected 403.");
```

Reply:
Resolved. Thanks